config/src/config.rs

```diff
@@ -589,6 +463,23 @@ impl Default for ConsensusConfig {
             contiguous_rounds: 2,
             max_pruned_blocks_in_mem: None,
             pacemaker_initial_timeout_ms: None,
+            consensus_keypair: ConsensusKeyPair::default(),
+            consensus_keypair_file: PathBuf::from("consensus_keypair.config.toml"),
+        }
+    }
+}
+
+#[cfg(any(test, feature = "testing"))]
```

Comment:
from https://github.com/libra/libra/blob/master/documentation/coding_guidelines.md#testing I think you can replace this manual `Clone` impl with

```rust
#[cfg_attr(any(test, feature = "testing"), derive(Clone))]
#[derive(Debug, Deserialize, Serialize)]
#[serde(default)]
pub struct ConsensusConfig {
    // ..
}
```